DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -132,6 +147,8 @@ protected override void Setup(DiscordClient client)
         this.AddCheck<RequireNsfwCheck>();
         this.AddCheck<RequirePermissionsCheck>();
         this.AddCheck<TextMessageReplyCheck>();
+
+        this.AddParameterCheck<RequireHierarchyCheck>();
```

Comment:
I still believe that we should be registering all checks within our own assembly instead of registering them all individually.

Reply:
i still believe that assembly discovery shouldn't exist in the first place: it's an easy source of bugs (maybe not so much for us, but still a possibility) and it's less flexible with regards to any other metadata (in this case especially DI scopes)